### PR TITLE
Feat: Revamp date picker (and create timepicker)

### PR DESCRIPTION
**Pull request by pKorsholm**
**What** 
- date picker based on `react-datepicker`
- custom time picker

**How** 
- implement radix popovers with input container and header for field styling
- adapt styling for `react-datepicker` to match design (using `@layer components` in global.css)
- create custom number roller component(see below)
- both components expose the same interface: an extension of `input-header` with `date:Date` and `onChange: (date:Date) => void`, which still conforms with the old implementation of the datepicker

image

image

### CODE REVIEW

**Review by zakariaelas — APPROVED** [review]
Awesome work @pKorsholm :rocket: !! Looks very very nice :heart_eyes: . Left tiny comments.

File: src/components/atoms/date-picker/date-picker.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+import ChevronLeftIcon from "../../fundamentals/icons/chevron-left-icon"
+import * as PopoverPrimitive from "@radix-ui/react-popover"
+import InputContainer from "../../fundamentals/input-container"
+import InputHeader from "../../fundamentals/input-header"
+import clsx from "clsx"
+import { DateTimePickerProps } from "./types"
+
+const DatePicker: React.FC<DateTimePickerProps> = ({
+  date,
+  onChange,
```
[the remark this answers is not in the document]
Comment (pKorsholm) [reply src/components/atoms/date-picker/date-picker.tsx:17]: Yeah i agree, i mostly kept the name for backwards compatilibity, but i think changing it to `onSubmitDate` is best

File: src/components/atoms/date-picker/time-picker.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          >
+            <InputContainer className="border-0 shadown-none focus-within:shadow-none">
+              <div className="w-full flex text-grey-50 pr-0.5 justify-between">
+                <InputHeader
+                  {...{
+                    label,
+                    required,
+                    withTooltip,
+                    tooltipText,
+                    tooltipProps,
```
[the remark this answers is not in the document]
Comment (pKorsholm) [reply src/components/atoms/date-picker/time-picker.tsx:61]: I have changed the pickers

File: src/components/fundamentals/icons/chevron-left-icon/index.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <svg
+      width={size}
+      height={size}
+      viewBox="0 0 16 16"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...attributes}
+    >
+      <path
```
Comment (kasperkristensen) [inline src/components/fundamentals/icons/chevron-left-icon/index.tsx:18]: Small change here -> change all kebab cased svg values to camel case, as this throws errors 😄

**Review by olivermrbl — APPROVED** [review]
LGTM, this is soooo cool @pKorsholm !!

**Review by kasperkristensen — APPROVED** [review]
LGTM 😄  Added some tiny nit comments

**Comment by olivermrbl** [comment]
Feel free to merge when ready 👍

**Pull request merged**